client/src/frontend/containers/ProjectCollections/index.js

```diff
@@ -145,9 +145,10 @@ export class ProjectsCollectionsContainer extends Component {
     return (
       <>
         <CheckFrontendMode debugLabel="ProjectCollections" isProjectSubpage />
-        <Utility.BackLinkPrimary
-          link={lh.link("frontendProjectsAll")}
-          backText={"Back to projects"}
+        <Breadcrumbs
+          breadcrumbs={[
+            { to: lh.link("frontendProjectsAll"), label: "Back to projects" }
+          ]}
```

Comment:
Is there a reason why the component is being rendered directly rather than using the breadcrumb context?